__tests__/app.test.js

```diff
@@ -199,3 +200,72 @@ describe('PATCH /api/review/:reviewid', () => {
     })
 });
 
+describe.only(' GET /api/reviews', () => {
+    test('200: responds with an array of reviews', () => {
+        return request(app)
+        .get('/api/reviews')
+        .expect(200)
+        .then( (response) => {
+            expect(typeof response.body).toBe('object')
+            expect(Array.isArray(response.body.reviews)).toBe(true)
+            expect(response.body.reviews.length > 0).toBe(true)
+            response.body.reviews.forEach((review) => {
+                expect(review).toHaveProperty('owner', expect.any(String))
+                expect(review).toHaveProperty('title', expect.any(String))
+                expect(review).toHaveProperty('review_id', expect.any(Number))
+                expect(review).toHaveProperty('category', expect.any(String))
+                expect(review).toHaveProperty('review_img_url', expect.any(String))
+                expect(review).toHaveProperty('created_at', expect.any(String))
+                expect(review).toHaveProperty('votes', expect.any(Number))
+                expect(review).toHaveProperty('designer', expect.any(String))
+                expect(review).toHaveProperty('comment_count')         
+            })
+        })
+    })
+    test('200: reviews are filtered by category query', () => {
+        return request(app)
+        .get('/api/reviews?category=dexterity')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.reviews.length > 0).toBe(true)
```

Comment:
Amend this expect statement, same feedback as above... make it a length test which checks that you are getting the exact number of reviews back which have dexterity as the category type.